cli/src/cli.rs

```diff
@@ -1324,11 +1315,28 @@ fn process_deploy(
         CliError::DynamicProgramError("Program account allocation failed".to_string())
     })?;
 
+    let (blockhash, _, _) = rpc_client
+        .get_recent_blockhash_with_commitment(config.commitment)?
+        .value;
+
+    let mut write_transactions = vec![];
+    for message in write_messages.into_iter() {
+        let mut tx = Transaction::new_unsigned(message.clone());
```

Comment:
NIT: You shouldn't need the `clone()` here since you're `into_iter()`ing, right?